### PR TITLE
[doc] Improve Pulsar security authorization

### PR DESCRIPTION
**Pull request by Monica-zy**
Motivation
Improve the language and the overall descriptive style of the Pulsar Security document (security authorization section): http://pulsar.apache.org/docs/en/next/security-authorization/

Modifications
Adjust the tone, personal pronouns, voice also some typo errors of some sentences in the document.

### CODE REVIEW

**Comment by sijie** [comment]
@Anonymitaet ^^

**Comment by Monica-zy** [comment]
run cpp tests

**Comment by Anonymitaet** [comment]
@Jennifer88huang since you are oncall this week, could you please help review? Thank you

File: site2/docs/security-authorization.md [hunk 1]
```diff
@@ -5,44 +5,42 @@ sidebar_label: Authorization and ACLs
...
 superUserRoles=my-super-user-1,my-super-user-2
 ```
 
-> A full list of parameters available in the `conf/broker.conf` file,
-> as well as the default values for those parameters, can be found in [Broker Configuration](reference-configuration.md#broker) 
+> A full list of parameters is available in the `conf/broker.conf` file.
+> You can also find the default values for those parameters in [Broker Configuration](reference-configuration.md#broker). 
 
-Typically, superuser roles are used for administrators and clients but also for broker-to-broker authorization. When using [geo-replication](concepts-replication.md), every broker needs to be able to publish to all the other clusters' topics.
+Typically, you can not only use superuser roles for administrators and clients but also for broker-to-broker authorization. When you are using [geo-replication](concepts-replication.md), every broker needs to be able to publish to all the other topics of clusters.
```
Comment (Jennifer88huang) [inline site2/docs/security-authorization.md]: When you use...

File: site2/docs/security-authorization.md [hunk 2]
```diff
@@ -5,44 +5,42 @@ sidebar_label: Authorization and ACLs
...
 
 ### Proxy Roles
 
-By default, the broker treats the connection between a proxy and the broker as a normal user connection. The user is authenticated as the role configured in ```proxy.conf``` (see ["Enabling TLS Authentication on Proxies"](security-tls-authentication#on-proxies)). However, this is rarely the behaviour that the user desires when connecting to the cluster through a proxy. The user expects to be able to interact with the cluster as the role for which they have authenticated with the proxy.
+By default, the broker treats the connection between a proxy and the broker as a normal user connection. The broker authenticates the user as the role configured in ```proxy.conf``` (see ["Enabling TLS Authentication on Proxies"](security-tls-authentication#on-proxies)). However, when the user is connecting to the cluster through a proxy, the user rarely desires the authentication. The user expects to be able to interact with the cluster as the role for which they have authenticated with the proxy.
```
Comment (Jennifer88huang) [inline site2/docs/security-authorization.md]: 1. if `proxy.conf` is a file name, use `proxy.conf`;
2. refine "when the user is connecting...." as "when the user connects...";
3. replace the word "desires".

File: site2/docs/security-authorization.md [hunk 3]
```diff
@@ -5,44 +5,42 @@ sidebar_label: Authorization and ACLs
...
 
-Pulsar uses *Proxy roles* to enable this. Proxy roles are specified in the broker configuration file, [`conf/broker.conf`](reference-configuration.md#broker). If a client that is authenticated with a broker is one of its ```proxyRoles```, all requests from that client must also carry information about the role of the client that is authenticated with the proxy. If this information, which we call the *original principal*, is missing, the client will not be able to access anything.
+Pulsar uses *Proxy roles* to enable this. Proxy roles are specified in the broker configuration file, [`conf/broker.conf`](reference-configuration.md#broker). If a client that is authenticated with a broker is one of its ```proxyRoles```, all requests from that client must also carry information about the role of the client that is authenticated with the proxy. If this information, which we call the *original principal*, is missing, the client is not able to access anything.
```
Comment (Jennifer88huang) [inline site2/docs/security-authorization.md]: re-check those sentences.

File: site2/docs/security-authorization.md [hunk 4]
```diff
@@ -5,44 +5,42 @@ sidebar_label: Authorization and ACLs
...
 
-Both the *proxy role* and the *original principle* must be authorized to access a resource for that resource to be accessible via the proxy. Administrators can take two approaches to this.
+You must authorize the *proxy role* and the *original principle* to access a resource for that resource to be accessible via the proxy. Administrators can take two approaches to this.
```
Comment (Jennifer88huang) [inline site2/docs/security-authorization.md]: improve this sentence.

File: site2/docs/security-authorization.md [hunk 5]
```diff
@@ -5,44 +5,42 @@ sidebar_label: Authorization and ACLs
...
 
-The more secure approach is to grant access to the proxy roles each time you grant access to a resource. For example, if you have a proxy role ```proxy1```, when a tenant is created by the superuser, ```proxy1``` should be specified as one of the admin roles. When a role is granted permissions to produce or consume from a namespace, if that client wants to produce or consume through a proxy, ```proxy1``` should also be granted the same permissions.
+The more secure approach is to grant access to the proxy roles each time you grant access to a resource. For example, if you have a proxy role ```proxy1```, when the superuser creats a tenant, you should specify ```proxy1``` as one of the admin roles. When a role is granted permissions to produce or consume from a namespace, if that client wants to produce or consume through a proxy, you should also grant ```proxy1``` the same permissions.
```
Comment (Jennifer88huang) [inline site2/docs/security-authorization.md]: Why use ```proxy1```?

File: site2/docs/security-authorization.md [hunk 6]
```diff
@@ -51,31 +49,31 @@ proxyRoles=my-proxy-role
 superUserRoles=my-super-user-1,my-super-user-2,my-proxy-role
 ```
 
-## Administering Tenants
+## Administer Tenants
```
Comment (Jennifer88huang) [inline site2/docs/security-authorization.md]: 1. Refine the title;
2. Add a summary/lead-in sentence between heading2 and heading3, or reorganize the information.
For guidelines, see "Headings" in IBM style guides.
image

File: site2/docs/security-authorization.md [hunk 7]
```diff
@@ -51,31 +49,31 @@ proxyRoles=my-proxy-role
...
 
-### Creating a new tenant
+### Create a new tenant
 
-A Pulsar [tenant](reference-terminology.md#tenant) is typically provisioned by Pulsar [instance](reference-terminology.md#instance) administrators or by some kind of self-service portal.
+Pulsar [instance](reference-terminology.md#instance) administrators or some kind of self-service portal typically provisions a Pulsar [tenant](reference-terminology.md#tenant). 
 
-Tenants are managed using the [`pulsar-admin`](reference-pulsar-admin.md) tool. Here's an example tenant creation command:
+You can manage tenants using the [`pulsar-admin`](reference-pulsar-admin.md) tool. Here's an example tenant creation command:
```
Comment (Jennifer88huang) [inline site2/docs/security-authorization.md]: Avoid to use "Here's ..." in technical writing.

**Comment by sijie** [comment]
@Jennifer88huang please review it again

**Comment by sijie** [comment]
run java8 tests

**Pull request merged**